test/transformations/extent.jl

```diff
@@ -0,0 +1,17 @@
+using Test
+ 
+import GeoInterface as GI
+import GeometryOps as GO
```

Comment:
Would be good to test at least one curve or something that isn't nested since that is a separate function

Reply:
If we use `apply` for everything than it also needs less tests.